app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -104,15 +104,50 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
     else
       StatusMessage.display 'failure', t "unsaved_changes_error"
 
+  $scope.getOrdersToCancel = (lineItemsToDelete) ->
```

Comment:
It looks like you replaced this logic and now you don't need this function any more. Is that correct?

Reply:
that's correct, sorry about that. Will remove later today